solutions/cses-1662.mdx

```diff
@@ -134,21 +134,17 @@ public class subarrayDivisibility {
 ```py 
 n = int(input())
 Z = map(int, input().split())
-
-M = [0] * n
-
+M = [0]*n
 psums = 0
 M[psums] = 1
 for a in Z:
-	psums += a
+	psums+= a
 	# Remember to account for negative sums
-	M[(psums % n + n) % n] += 1
-
+	M[(psums%n+n)%n]+= 1
```

Comment:
what why did you do this

Reply:
just to reduce the space !

Reply:
that's not what envy meant by the spaces
he meant the ones like
```py
x = [ 0 ]

Reply:
oh ok
sholud I need to change it and update again ?

Reply:
oh ok
should  I need to change it and update again ?

Reply:
Please let me know, so that I can update and add again

Reply:
@SansPapyrus683 should I need to update again ?

Reply:
yeah, add the spaces back

Reply:
added required  spaces @SansPapyrus683